beluga_demo_nav2/launch/beluga_demo_nav2.launch.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2024 Ekumen, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This is all-in-one launch script intended for use by nav2 developers.
```

Comment:
@Alondruck is this inspired or a verbatim copy? Nav2 has its own copyrights and licenses we have to be mindful about.

Reply:
Yes, is a copy with some changes. What do I need to do is this cases?